code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -129,8 +134,27 @@
 	else
 		Tempstun = 0
 
+	if(stat != DEAD)
+		var/co2_percentage =0
+		if(("co2" in environment.gases) && environment.total_moles())
```

Comment:
I can add a zero moles of co2 to gas mixture. Then "co2" would exist, but total_moles would be empty, causing division by zero.